app/main.py

```diff
@@ -1,2 +1,66 @@
+from typing import Any, Optional
```

Comment:
The use of '|' for type hints is only supported in Python 3.10 and later. If compatibility with earlier versions is needed, consider using 'Optional[int]' instead of 'int | None'.